src/helpers/helpers.ts

```diff
@@ -90,6 +90,19 @@ const getTimeDisplayHowLongAgo = (timestamp: MomentInput, { devMode = DEV_MODE }
  */
 const formatDate = (date: Date) => moment.utc(date).format('DD MMMM Y, h:mm A z');
 
+/**
+ * Normalizes hosts textarea content into array that can be submitted to backend.
+ *
+ * @param {string} data - host textarea content.
+ * @returns Array of host values which will be submitted to backend.
+ */
+const normalizeHosts = (data: string) => {
+  return data
+    .replaceAll(/\\n|\\r|\s/g, ',')
+    .split(',')
+    .filter(host => !!host || host.length > 0);
```

Comment:
require: filter provides redundant checks any values that come back as an empty string should be `falsey` meaning we could just use `host => !!host);`

...but we've been avoiding the double bang since it's hard to read and it's still a hack that's only made its way back in because of newer js/ts... we've been using...

```
...
    .replaceAll(/\\n|\\r|\s/g, ',')
    .split(',')
    .filter(Boolean);
```

optional
you may also consider replacing `\s` too, and/or using something like the JS string `.trim()` before doing your replace

Reply:
Everything done in 7306a293deedf400f2cd47cefb350c8a44a32b7e